qutebrowser/mainwindow/tabwidget.py

```diff
@@ -139,6 +139,7 @@ def update_tab_title(self, idx):
         fields = self.get_tab_fields(idx)
         fields['title'] = fields['title'].replace('&', '&&')
         fields['index'] = idx + 1
+        fields['private'] = ' [Private Mode] ' if tab.private else ''
```

Comment:
I wasn't sure, so that's why I added it. I have removed it now and pushed an updated commit.